geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/Transformers.scala

```diff
@@ -489,13 +514,32 @@ class GeometryFunctionFactory extends TransformerFunctionFactory {
     }
   }
 
+  val multiPolygonParserFn = TransformerFn("multipolygon") { args =>
```

Comment:
wouldn't these all be handled by the plain 'geometry' function? unless we're doing some special processing like we do with points where we accept (x, y)... maybe instead we should just alias 'geometry' to all the other types. Or if we're planning to eventually expand the parsing options here I'm ok with having them separate.

Reply:
i guess they have the advantage of failing if the geometry type is not the expected one